R/utils.R

```diff
@@ -457,22 +457,25 @@ valid_seq <- function(params) {
   }
 }
 
-resolve_prefix <- function(prefix, environment_var, base_pathname) {
+resolvePrefix <- function(prefix, environment_var, base_pathname) {
```

Comment:
Just wondering why this function is changed to camelCase when many other functions are snake_case.

Reply:
I'm going to open a PR to migrate all functions to camelCase and variables to snake_case, which follows the generally accepted style rules for R. That will likely occur within the next week, prior to CRAN submission.

Reply:
cf. https://google.github.io/styleguide/Rguide.html, though they use Pascal case instead of camel case, and maybe we should also. 🤔

Reply:
Kk, thx. Pick one, enforce it. Move on :)

Reply:
:neckbeard: 📘 🤺